pkg/controller/v1beta1/inferenceservice/components/predictor.go

```diff
@@ -276,6 +405,33 @@ func (p *Predictor) Reconcile(isvc *v1beta1.InferenceService) (ctrl.Result, erro
 		return !utils.Includes(constants.ServiceAnnotationDisallowedList, key)
 	})
 
+	// Autoscaler should be ignored when multiNodeEnabled is true
+	workerObjectMeta = metav1.ObjectMeta{}
+	if multiNodeEnabled {
+		annotations[constants.AutoscalerClass] = string(constants.AutoscalerClassExternal)
```

Comment:
IMO, if the autoscaler annotation was explicitly set by the user, rather than ignoring, we should finish with error and provide feedback to the user that multi-node does not support autoscalers.

Reply:
updated.